doxygen failed to install (10.5/G5)

https://gist.github.com/87907820afa034d9e59f

```
-- Performing Test ICONV_COMPILES
-- Performing Test ICONV_COMPILES - Failed
-- Could NOT find ICONV (missing:  ICONV_COMPILES) 
CMake Error at cmake/FindIconv.cmake:121 (MESSAGE):
  Unable to determine iconv() signature
Call Stack (most recent call first):
  CMakeLists.txt:64 (find_package)
```

Do you mind redoing the build with `-d` and fetching a copy of the logs referenced here?

```
-- Configuring incomplete, errors occurred!
See also "/tmp/doxygen20151226-67158-9y7ikq/doxygen-1.8.10/build/CMakeFiles/CMakeOutput.log".
See also "/tmp/doxygen20151226-67158-9y7ikq/doxygen-1.8.10/build/CMakeFiles/CMakeError.log".
```

Same type of problem on 1.8.14:

-- Performing Test ICONV_COMPILES
-- Performing Test ICONV_COMPILES - Failed
-- Could NOT find ICONV (missing:  ICONV_COMPILES) 
CMake Error at cmake/FindIconv.cmake:130 (MESSAGE):
  Unable to determine iconv() signature
Call Stack (most recent call first):
  CMakeLists.txt:92 (find_package)

--- a/Library/Formula/doxygen.rb
+++ b/Library/Formula/doxygen.rb
@@ -27,6 +27,9 @@ class Doxygen < Formula
   depends_on "qt" if build.with? "doxywizard"
   depends_on "llvm" => "with-clang" if build.with? "libclang"
 
+  # /Developer/SDKs/MacOSX10.4u.sdk/usr/include/stdarg.h:4:25: error: stdarg.h: No such file or directory
+  fails_with :gcc if MacOS.version == :tiger
+
   def install
     # This flag was introduced after GCC 4.2.
     # This is necessary on Tiger, since we don't have superenv yet.

--- a/Library/Formula/doxygen.rb
+++ b/Library/Formula/doxygen.rb
@@ -28,6 +28,12 @@ class Doxygen < Formula
   depends_on "llvm" => "with-clang" if build.with? "libclang"
 
   def install
+    # This flag was introduced after GCC 4.2.
+    # This is necessary on Tiger, since we don't have superenv yet.
+    if [:gcc, :gcc_4_0].include? ENV.compiler
+      inreplace "CMakeLists.txt", "-Wno-deprecated-register", ""
+    end
+
     args = std_cmake_args
     args << "-Dbuild_wizard=ON" if build.with? "doxywizard"
     args << "-Duse_libclang=ON -DLLVM_CONFIG=#{Formula["llvm"].opt_bin}/llvm-config" if build.with? "libclang"

